third_party/build/build-dist.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+# Build a stubs-only distribution for a third-party package.
+#
+# Usage: python3.7 build-dist.py <PACKAGE>
+#
+# The resulting wheel will be saved in third-party/build/dist.
+
+import datetime
+import os
+import re
+import shutil
+import subprocess
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List
+
+DIST_SUFFIX = "-ts"
```

Comment:
I find it amusing this looks like typescript :)

Reply:
Actually, when I reviewed my suggestion before writing this script I wondered why I had chosen that suffix and whether I made a mistake because I had written a lot of typescript lately. Then I remembered that it was the abbreviation for typeshed. :)

That said, I think we should get the pypi maintainers on board for a final decision for a suffix.